pontoon/base/templates/base.html

```diff
@@ -29,6 +29,12 @@
       {% endblock %}
     {% endblock %}
 
+    <!-- Import Chart.js and Moment.js from static files -->
+    <script language="javascript" type="text/javascript" src="{{ static('js/lib/chart.umd.min.js') }}"></script>
+    <script language="javascript" type="text/javascript" src="{{ static('js/lib/moment.min.js') }}"></script>
+    <script language="javascript" type="text/javascript" src="{{ static('js/lib/chartjs-adapter-moment.min.js') }}"></script>
```

Comment:
This will load all these libs on most of the views, even when they aren't needed.

We should load them, like other static files, through Django pipeline in settings, and only in the view where Chart.js was previously used.

Also, seems like pontoon/base/static/js/lib/chart.umd.js.map is not used?

Reply:
You're right, I was getting a console error when I didn't include this file through base.html, but now that I've shifted over to local imports it seems this file isn't necessary.